docs/source/guides/libraries.rst

```diff
@@ -67,12 +67,256 @@ Python library
 Where to get
 ^^^^^^^^^^^^
 
+Prerequirements:
+
+- cmake(3.11 or higher)
+- git
+- g++
+- boost(1.65 or higher, only system and filesystem)
+- swig(3.0.12 can be built --without-pcre)
+- protobuf
+
+Install iroha python libraries:
+
+- Through pip
+
+    .. code:: sh
+
+      pip install iroha
+
+    For the latest version
+
+    .. code:: sh
+
+      pip install -i https://testpypi.python.org/pypi iroha
+
+- Source code
+
+    .. code:: sh
+
+      git clone https://github.com/hyperledger/iroha.git
+      cd iroha
+
+    For the latest version checkout to develop branch
+
+    .. code:: sh
+
+      git checkout develop
+
+    .. code:: sh
+
+      cmake -H. -Bbuild -DSWIG_PYTHON=ON -DSHARED_MODEL_DISABLE_COMPATIBILITY=ON -DSUPPORT_PYTHON2=ON;
+      cmake --build build -- -j4
+
+    After this you can find iroha python library in **iroha/build/shared_model/bindings** folder.
+
+Install iroha protobuf files:
```

Comment:
Why do we need that? Please explain in a short comment or something

Reply:
It is wrong naming. You compile .py classes from protobuf schema using proton compiler. It is not an installation.